src/FixedDataTableCell.js

```diff
@@ -277,11 +260,26 @@ class FixedDataTableCell extends React.Component {
       );
     }
 
-    var cellProps = {
-      columnKey,
-      height,
-      width
-    };
+    var cellProps;
+    if(!this.props.isHeader) {
+      cellProps = {
+        columnKey,
+        height,
+        width,
+      }
+    }
+    else {
+      cellProps = {
+        columnKey,
+        height,
+        width,
+        tableHeight: this.props.tableHeight,
```

Comment:
since tableHeight is a new prop, add it to proptypes

Reply:
React throws an error `_ not defined`. But we can do it another way:
```
var cellProps = {
      columnKey,
      height,
      width,
    };
    if (this.props.isHeader) {
      cellProps = {
        ...cellProps,
        tableHeight: this.props.tableHeight,
        left: this.props.left,
        cellGroupLeft: this.props.cellGroupLeft,
        touchEnabled: this.props.touchEnabled,
        isRTL: this.props.isRTL,
        scrollX: this.props.scrollX,
        isFixed: this.props.isFixed,
        availableScrollWidth: this.props.availableScrollWidth,
        maxScrollX: this.props.maxScrollX,
        groupColumnWidths: this.props.groupColumnWidths,
        _scrollToX: this.props._scrollToX,
        onHorizontalScroll: this.props.onHorizontalScroll,
      };
    }
```
Would this be called clean?